### ISSUE TITLE
[BUG]Error: could not open `libraries/net/minecraftforge/forge/1.12.2-14.23.5.2860/unix_args.txt'

### ISSUE THREAD

**Issue by X3bex** [thecoder-001/MineColab#100]
**Describe the bug**
Hello, I get this error when I start the server: 
apt cache successfully updated
Failed to install OpenJdk16.
Openjdk17 doesn't seems to be installed or isn't working, falling back to java 11 . You might experience reduced performance. Minecraft 1.17 and above might fail to launch.
Drive already mounted at /content/drive; to attempt to forcibly remount, call drive.mount("/content/drive", force_remount=True).
/content/drive/My Drive/Minecraft-server
colabconfig.json  forge-1.12.2-14.23.5.2860.jar  libraries		      server.jar
eula.txt	  installer.log			 minecraft_server.1.12.2.jar
Procedding to use ngrok
Get your authtoken from https://dashboard.ngrok.com/auth
Authtoken saved to configuration file: /root/.config/ngrok/ngrok.yml
Your server address is 0.tcp.sa.ngrok.io:13726
Starting server...
Error: could not open `libraries/net/minecraftforge/forge/1.12.2-14.23.5.2860/unix_args.txt'

**To Reproduce**

1. Go to the first cell
2. Click on 'start'
3. See error

**Minecraft server Version**
1.12.2

**Comment by N-aksif-N** [comment]
Your server is using Java 17


1.12.2 requires Java 8 to work

**Comment by X3bex** [comment]
> Your server is using Java 17
> 
> 1.12.2 requires Java 8 to work
no, I'm using java 8 and it just appears that it doesn't load, I need if someone knows about that error at the end

**Comment by N-aksif-N** [comment]
but you dont set it to default. some code will have you:

# Java 8 is required to run Minecraft versions 1.12 through 1.17. Java 17 is required to run Minecraft version 1.18 and up.
if version > "1.17":
    ! sudo apt install openjdk-17-jre-headless &>/dev/null &&echo "Openjdk17 successfully installed" || echo "Failed to install OpenJdk17."
    # Set default to java 17. Same goes to java8
    environ["JAVA_HOME"] = "/usr/lib/jvm/java-17-openjdk-amd64"
    ! update-alternatives --set java /usr/lib/jvm/java-17-openjdk-amd64/bin/java
else:
    ! sudo apt install openjdk-8-jre-headless &>/dev/null &&echo "Openjdk8 successfully installed" || echo "Failed to install OpenJdk8."
    environ["JAVA_HOME"] = "/usr/lib/jvm/java-8-openjdk-amd64"
    ! update-alternatives --set java /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java

**Comment by N-aksif-N** [comment]
Hey, why not look at my page: https://github.com/N-aksif-N/Minecolab
Maybe, you will be interested in that. Remember to download it.

**Comment by PabloDevh** [comment]
How would it look implemented?
sorry for my bad english

**Comment by N-aksif-N** [comment]
First, It will check whether the server is running on version upper and in 1.17 or below 1.17. After that, it will install the java according to the version of server. Then, it will set the environment and update-alternatives. These activities will set the default java, because the default java is always set on java 11.
